### ISSUE TITLE
JS Intellisense not detecting ES6 Symbol properties assigned after Initialization

### ISSUE THREAD

**Issue by MatthewPinheiro** [microsoft/vscode#119333]
- VS Code Version: 1.54.3
- OS Version: Windows 10 Pro, version 20H2, build 19042.804

Steps to Reproduce:

1. Initialize a Symbol
2. Initialize an empty object literal
3. Assign the object a function, using the symbol as its property

The following code also demonstrates some of the steps to reproduce:

```
const symKey = Symbol();


/** @namespace */
var objLiteral = {
    literalPublicFn1() {},

    /**
     * You found the secret fn!
     */
    [symKey]() {},
};


/** @namespace */
var objAssigned = {};

/** @memberof objAssigned */
objAssigned.laterPublicFn1 = function() {};

/** 
 * You found the secret fn!
 * @memberof objAssigned 
 * */
objAssigned[symKey] = function () {}
```

For both objects, the string properties are picked up by Intellisense and autocompleted as expected. However, Intellisense only provides function info for the symbol on the `objLiteral` object, where the symbol property was declared as part of the literal. For the `objAssigned` object, where the symbol property was assigned after initialization, no function info is provided.

In other words, intellisense works when calling `objLiteral[symKey]()` but not when calling `objAssigned[symKey]()`

Does this issue occur when all extensions are disabled?: Yes
